build.sbt

```
@@ -38,12 +38,12 @@ sonatypeCredentialHost := "s01.oss.sonatype.org"
 
 val Version = new {
   object CE3 {
-    val fs2        = "3.2.12"
-    val cats       = "3.3.14"
+    val fs2        = "3.2.14-75-7902cbf"
+    val cats       = "3.3.14-1-5d11fe9"
```

```suggestion
    val cats       = "3.3.14"
```

https://github.com/typelevel/cats-effect/issues/1302#issuecomment-1247387034